snuba/settings_base.py

```diff
@@ -7,10 +8,23 @@
 
 PORT = 1218
 
+DEFAULT_DATASET_TYPE = 'events'
+DATASETS = {
```

Comment:
How does this get overridden for the production dataset ?

Reply:
on prod machines, [this](https://github.com/getsentry/ops/blob/master/cookbooks/snuba/templates/default/prod_settings.py.erb) file gets put in there. It will have to change after this release. Should be able to replace most of those settings with just the dataset classnames.